tests/test_api.py

```diff
@@ -1,15 +1,14 @@
 """Test Api class"""
+import apispec
```

Comment:
Why move this up there?

Reply:
`A` is a first letter in alphabet. So I guess I've pressed my shortcut for `isort` :). Muscle memory. I will revert it back.

Reply:
OK. Turns out it is what `pre-commit` does on it's own.